src/lib/runtime/Web3FunctionRunnerPool.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { StaticJsonRpcProvider } from "@ethersproject/providers";
+import { jest } from "@jest/globals";
+import path from "path";
+
+import { Web3FunctionBuilder } from "../builder";
+import {
+  Web3FunctionContextData,
+  Web3FunctionRunnerOptions,
+  Web3FunctionVersion,
+} from "../types";
+import { Web3FunctionRunnerPool } from "./Web3FunctionRunnerPool";
+
+const MAX_RPC_LIMIT = 100;
+const MAX_DOWNLOAD_LIMIT = 10 * 1024 * 1024;
+const MAX_UPLOAD_LIMIT = 5 * 1024 * 1024;
+const MAX_REQUEST_LIMIT = 110;
+const MAX_STORAGE_LIMIT = 1024; // kb
+
+describe("Web3FunctionRunnerPool", () => {
+  const LOCAL_BASE_PATH = path.join(process.cwd(), "src/lib/runtime/__test__");
```

Comment:
same path issue

Reply:
fixed in ccaf9abedf6194822325974f3a756c6cd8d2c1db